db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+ArtType.create([{
+  name: "visual art",
+  name: "writing",
+  name: "music",
+  name: "dance",
+  name: "theater",
+  name: "filmmaking"
+  }])
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.